tests/e2e/tests-library/WorkspaceHandlingTests.ts

```diff
@@ -73,6 +73,13 @@ export class WorkspaceHandlingTests {
 		await this.browserTabsUtil.waitAndSwitchToAnotherWindow(WorkspaceHandlingTests.parentGUID, TIMEOUT_CONSTANTS.TS_IDE_LOAD_TIMEOUT);
 	}
 
+	async createAndOpenWorkspaceWithExistedWorkspaceName(stack: string): Promise<void> {
+		Logger.debug('create and open workspace with existed workspace name.');
```

Comment:
There is no need to enter method description like `'create and open workspace with existed workspace name.'` when call Logger methods after commit https://github.com/eclipse/che/commit/df80cc80dcf00321cf839278cd516aed5b6efc87